### DISCUSSION TITLE
DomainAdmin Role Can see all guest network.

### DISCUSSION THREAD

**Discussion by j3cky** [apache/cloudstack#9474] [Q&A] [answered]
Hi,
i have something to ask, after i upgrade my cloudstack server from 4.19.0 to 4.19.1, as domainadmin role type i can see all guest network from other account even admin account guest network. This is not happened in version 4.19.0.
is it expected  with newer version?is it possible to hide other guest network which in not belong to the account?

**Comment by weizhouapache** [comment, marked as answer]
@j3cky 
we are working on a security release for it
#9456
